### ISSUE TITLE
ECS.7 Use arrows to navigate through card previews

### ISSUE THREAD

**Issue by ashkjones** [PonderTeam/ponder-app#35]
## User Story 
As a user, I want buttons on the card preview, so that I can easily navigate to the cards I want to add in my sequence

## Sub-tasks 
- [ ] Arrows icon in the card preview UI
- [ ] When arrows icon clicked, card preview is updated with new card term.
- [ ] Highlighted Term in selector is updated

## Notes
- depends on #31 
- Do this after #36
